students.html

```diff
@@ -0,0 +1,148 @@
+---
```

Comment:
I'm not removing them completely, just that a list will be shown on the main page, also the separate file is part of the site which can easily be accessed from the navigation bar.